### ISSUE TITLE
Question: KPL Disable Retries

### ISSUE THREAD

**Issue by czegarram** [awslabs/amazon-kinesis-producer#432]
Hi, thanks for the amazing work with KPL, it's solving a lot of issues in the projects I'm implementing. 

I have a new feature to implement when I produce data: the uniqueness of the records.

I noticed that KPL retries requests event when failIfThrottled = true. Is there any way to avoid tries when a request fails for any reason?

**Comment by czegarram** [comment]
Version 0.14.10
SO: Mac OS Big Sur

**Comment by VladimirPchelko** [comment]
No, just use java kinesis client and amazon-kinesis-aggregator for purposes of better behavior control.

Also don't use KPL in AWS lambdas since KPL starts a native process!!! - which is overkill!

But I'm not sure about the latest versions of these libraries, especially the 2nd version of aws sdk.
